internal/expr/parser.go

```diff
@@ -239,7 +260,7 @@ func (p *Parser) skipStringLiteral() (bool, error) {
 
 		// Reached end of string and didn't find the closing quote
 		cp.restore()
-		return false, fmt.Errorf("column %d: missing closing quote in string literal", cp.pos)
+		return false, fmt.Errorf("line %d, column %d: missing closing quote in string literal", p.lineNum, p.colNum())
```

Comment:
I created a helper function so as to reuse the message. Defering it to the Parse() function is more challenging because the errors would have to carry the line and column information.